### PR TITLE
update changelog with Unity XT known issue

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Added known issues for CSI Unity XT Driver

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/885 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by rajendraindukuri — DISMISSED** [review]

**Review by satyakonduri — DISMISSED** [review]
LGTM.

**Review by rajendraindukuri — APPROVED** [review]

**Review by chimanjain — DISMISSED** [review]

**Review by rensyct — APPROVED** [review]

File: CHANGELOG/CHANGELOG-1.8.0.md [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 ## Known Issues
 
-### CSI PowerStore: Not able to create ephemeral pods in OpenShift 4.13
+### CSI PowerFlex, CSI PowerStore, CSI Unity XT: Not able to create ephemeral pods in OpenShift 4.13
```
Comment (shaynafinocchiaro) [inline CHANGELOG/CHANGELOG-1.8.0.md:19]: Please update the table of contents with new heading

File: CHANGELOG/CHANGELOG-1.8.0.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ In OpenShift 4.13, the root user is not allowed to perform write operations on N
 
 The workaround for this issue is to disable root squashing by setting allowRoot: "true" in the NFS storage class.
 
+### CSI Drivers: Volume limit for pending PVCs is not obeyed if the volume limit is exhausted and the CSI Driver restarts
```
Comment (shaynafinocchiaro) [inline CHANGELOG/CHANGELOG-1.8.0.md:31]: Same as above

File: CHANGELOG/CHANGELOG-1.8.0.md (hunk 1 above)
Comment (panigs7) [reply CHANGELOG/CHANGELOG-1.8.0.md:20]: Done

File: CHANGELOG/CHANGELOG-1.8.0.md (hunk 2 above)
Comment (panigs7) [reply CHANGELOG/CHANGELOG-1.8.0.md:32]: Done

File: CHANGELOG/CHANGELOG-1.8.0.md (hunk 1 above)
Comment (panigs7) [reply CHANGELOG/CHANGELOG-1.8.0.md:20]: Done

File: CHANGELOG/CHANGELOG-1.8.0.md (hunk 2 above)
Comment (panigs7) [reply CHANGELOG/CHANGELOG-1.8.0.md:32]: Done

**Review by shaynafinocchiaro — APPROVED** [review]

**Pull request merged**